lib/generators/decidim/templates/initializer.rb

```diff
@@ -6,8 +6,8 @@
   config.mailer_sender = "change-me@domain.org"
   config.authorization_handlers = [ExampleAuthorizationHandler]
 
-  # Uncomment this lines to set your preferred locales
-  # config.available_locales = [:en, :ca, :es]
+  # Change this line to set your preferred locales
+  config.available_locales = [:en, :ca, :es]
```

Comment:
You mean using the hardcoded list for the organizations locales, but leaving the full list enabled here? As long as I don't have to fill in 8 locales for each field on every form, sounds good to me! :)